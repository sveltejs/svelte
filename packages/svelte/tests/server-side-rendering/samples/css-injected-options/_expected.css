
	.foo.svelte-okauro {
		color: red;
	}
