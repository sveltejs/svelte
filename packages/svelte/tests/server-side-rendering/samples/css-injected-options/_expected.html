<div class="foo svelte-okauro">foo</div>
